src/main/sensors/gyro.c

```diff
@@ -1150,3 +1190,44 @@ uint8_t gyroReadRegister(uint8_t whichSensor, uint8_t reg)
     return mpuGyroReadRegister(gyroSensorBusByDevice(whichSensor), reg);
 }
 #endif // USE_GYRO_REGISTER_DUMP
+
+#ifdef USE_DYN_LPF
+void dynLpfGyroUpdate(float throttle)
+{
+    if (dynLpfFilter == DYN_LPF_PT1 || dynLpfFilter == DYN_LPF_BIQUAD) {
+        uint16_t cutoffFreq = dynLpfMin;
+        if (throttle > dynLpfIdle) {
+            const float dynThrottle = (throttle - (throttle * throttle * throttle) * 0.333f) * 1.5f;
```

Comment:
Is this meant to be `0.333f`, or 1/3? If the latter, then `/ 3.0f` is a better way to write it, and optimiser will take care of converting the division by a constant into a (faster) multiplication by its inverse.